frontend/sozisel-app/src/components/Files/PresenterSessionFiles/PresenterSessionFiles.tsx

```diff
@@ -98,7 +98,9 @@ export function PresenterSessionFiles({
     return (
       <Dialog onClose={onClose} open={open} fullWidth maxWidth="md">
         <div className="PresenterSessionFiles">
-          <CircularProgress />
+          <div className="Circular">
```

Comment:
There is material ui class called `Center` as far as I remember and it probably does the same stuff.

Reply:
There is no such component as `Center` in Material UI, and I found this solution https://stackoverflow.com/questions/50766693/how-to-center-a-component-in-mui-and-make-it-responsive to make this component centered using Material UI components, but It seems to me that if there is no single component that centers the child without specific props properties, then my solution is just as good.

Reply:
Dobra, powaliło mi się z flutterem. Good enough!

Reply:
xd